luxtronik/calculations.py

```diff
@@ -264,18 +264,18 @@ def __init__(self):
             229: Celsius("ID_WEB_Temperatur_BW_oben"),
             230: HeatpumpCode("ID_WEB_Code_WP_akt_2"),
             231: Frequency("ID_WEB_Freq_VD"),
-            232: Unknown("Unknown_Calculation_232"),
-            233: Unknown("Unknown_Calculation_233"),
+            232: Celsius("Vapourisation_Temperature"),
+            233: Celsius("Liquefaction_Temperature"),
             234: Unknown("Unknown_Calculation_234"),
             235: Unknown("Unknown_Calculation_235"),
-            236: Unknown("Unknown_Calculation_236"),
-            237: Unknown("Unknown_Calculation_237"),
-            238: Unknown("Unknown_Calculation_238"),
-            239: Unknown("Unknown_Calculation_239"),
-            240: Unknown("Unknown_Calculation_240"),
-            241: Percent2("Circulation_Pump"),
-            242: Unknown("Unknown_Calculation_242"),
-            243: Unknown("Unknown_Calculation_243"),
+            236: Frequency("ID_WEB_Freq_VD_Soll"),
+            237: Frequency("ID_WEB_Freq_VD_Min"),
+            238: Frequency("ID_WEB_Freq_VD_Max"),
+            239: Kelvin("VBO_Temp_Spread_Soll"),
+            240: Kelvin("VBO_Temp_Spread_Ist"),
+            241: Percent2("HUP_PWM"),
+            242: Kelvin("HUP_Temp_Spread_Soll"),
+            243: Kelvin("HUP_Temp_Spread_Ist"),
```

Comment:
Wondering what this means physically? Don't remember to have seen this in the menu (will check later). But `HUP` means `Heizungsumwälzpumpe`. To my knowledge it is just circulating the water in the heating system. Not entirely really sure what `Soll` and `Ist` are supposed to mean in this context?

How exactly are those parameters called in the menu?

Reply:
These values can be seen in the (new) webinterface of the heat pump if you are logged in as Kundendienst.

If I understand my heat pump correctly, it only has two pumps: One for pumping the water into the bore holes (this is `VBO` which seems to come from `Ventilator, Brunnen- oder Soleumwälzpumpe`) and another one for pumping the heated water either into the hot water deposit or into the underfloor heating pipes).

These values report the `Ist` and `Soll` values of the temperatures of inflow and outflow. I think they are used to control the operating level of the pump.